term_proposal/titaneric/proposal.md

```diff
@@ -0,0 +1,52 @@
+# Basic neural network library supported auto-differentiation
+
+## Motivation
+
+Most deep learning courses aim to teach math behind the network, architecture and their applications.
+
+However, seldom course talk about how to implement the deep learning library and start every from scratch.
+
+Wish to implement this kind of library and learn how to design a better numerical software.
```

Comment:
> ... better numerical software.

Could you elaborate more about what do you mean by "better" here?  You can update this PR to add the elaboration in the proposal.  I think it will help you to define a more tangible goal.